cardinal/world.go

```diff
@@ -184,6 +186,7 @@ func NewWorld(opts ...WorldOption) (*World, error) {
 
 	// Register internal plugins
 	world.RegisterPlugin(newPersonaPlugin())
+	world.RegisterPlugin(world.SystemDelayManager)
```

Comment:
I thought this was ok because it's internal. Internal stuff is beneath the layer that requires the abstraction contract. The contract itself in my mind is between the User and the Developer and this contract isn't violated when it's used internally. 

It's not a huge deal for me I can remove this logic that uses the plugin and call stuff directly.